tests/UnifiedSpecTests/Util.php

```diff
@@ -134,8 +134,8 @@ final class Util
             'delete' => ['id'],
             'downloadByName' => ['filename', 'revision'],
             'download' => ['id'],
-            'uploadWithId' => ['id', 'filename', 'source', 'chunkSizeBytes', 'metadata'],
-            'upload' => ['filename', 'source', 'chunkSizeBytes', 'metadata'],
+            'uploadWithId' => ['id', 'filename', 'source', 'chunkSizeBytes', 'disableMD5', 'metadata'],
```

Comment:
Consider a comment like "disableMD5 is unsupported by PHPLIB but still defined in the spec" above these two lines.

Reply:
```suggestion
            // "disableMD5" is ignored but allowed for backward compatibility
            'uploadWithId' => ['id', 'filename', 'source', 'chunkSizeBytes', 'disableMD5', 'metadata'],
```